frontend/src/pages/AccountSent.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <q-page padding>
+    <h2 class="page-title">{{ $t('AccountSendTable.sent-funds') }}</h2>
+    <!-- User has not connected wallet  -->
+    <div v-if="!userAddress">
+      <p class="text-center">{{ $t('AccountSent.connect-your-wallet') }}</p>
+      <div class="row justify-center">
+        <connect-wallet>
+          <base-button class="text-center" :label="$t('AccountSent.connect-wallet')" />
+        </connect-wallet>
+      </div>
+    </div>
+    <div v-else class="q-mx-auto" style="max-width: 800px">
+      <account-send-table :sendMetadata="sendMetadata" />
+    </div>
+  </q-page>
+</template>
+
+<script lang="ts">
+import { defineComponent, ref } from 'vue';
+import useWallet from 'src/store/wallet';
+import useWalletStore from 'src/store/wallet';
```

Comment:
These are the same import with different names, let's only keep the `useWalletStore` one